CMakeLists.txt

```diff
@@ -164,6 +160,12 @@ target_compile_definitions(${PROJECT_NAME} PUBLIC
 	>
 )
 
+if("${GODOT_FLOAT_PRECISION}" STREQUAL "double")
+	target_compile_definitions( ${PROJECT_NAME} PUBLIC
+		REAL_T_IS_DOUBLE
+	)
+endif()
```

Comment:
Is this needed anymore now that we have this code?

```cmake
set(GODOT_PRECISION "single" CACHE STRING "Set the floating-point precision level (single|double)")
if ("${GODOT_PRECISION}" STREQUAL "double")
Oct 22, 2023
[SCons: Disable C++ exception handling by default](https://github.com/godotengine/godot-cpp/commit/bf1c03ab5f079d0931cf4390e7435a20cfaacfbd)
	add_definitions(-DREAL_T_IS_DOUBLE)
endif()
```

Reply:
I think you're right, this is no longer needed